### ISSUE TITLE
Blurred PopUp

### ISSUE THREAD

**Issue by inesmrad** [idurar/idurar-erp-crm#959]
**Describe the bug**
When clicking on the rocket icon on the header's left side, the pop up that surges isn't stable , it keeps moving up and down quickly in a blurry way , thus we can't read its content.
**To Reproduce**
1. sign up
2. log in 
3. click on the rocket icon on the header 
4. See the bug

**Expected behavior**
When clicking on the rocket icon the pop up should be stable for us to read its content

**Screenshots**

https://github.com/idurar/idurar-erp-crm/assets/133030569/01430f10-c10b-4b69-be76-ea47c41ea152

**Desktop (please complete the following information):**
 - OS:  64-bit version of Windows
 - Browser :Chrome
 - Version : 121.0.6167.161

**Comment by garvitmaloo** [comment]
Assign this to me 🙋🏽‍♂️

**Comment by inesmrad** [comment]
> Assign this to me 🙋🏽‍♂️
Sure you can work on it but i think only maintainers can assign you to an issue

**Comment by inesmrad** [comment]
> Assign this to me 🙋🏽‍♂️

Sure you can work on it, but i think only maintainers can assign you to an issue

**Comment by Siddhant-Kashyap** [comment]
can i work on this issue

**Comment by inesmrad** [comment]
> can i work on this issue

yeah sure

**Comment by Siddhant-Kashyap** [comment]
> > can i work on this issue
> 
> yeah sure

can you help me sign up issue 
![image](https://github.com/idurar/idurar-erp-crm/assets/86836609/c9d237ff-8535-432e-983d-058cb627b44f)
why its telling me 409 error (idurar_registration_allowed)
and how to tackle that

**Comment by inesmrad** [comment]
> > > can i work on this issue
> > 
> > 
> > yeah sure
> 
> can you help me sign up issue ![image](https://private-user-images.githubusercontent.com/86836609/305648183-c9d237ff-8535-432e-983d-058cb627b44f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgxODAwNjAsIm5iZiI6MTcwODE3OTc2MCwicGF0aCI6Ii84NjgzNjYwOS8zMDU2NDgxODMtYzlkMjM3ZmYtODUzNS00MzJlLTk4M2QtMDU4Y2I2MjdiNDRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE3VDE0MjI0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ0MjUzZTY1YTE1NmZkYzA3ZWY4NWFhZWIwOTZiODE3Y2NlOGVkOTczYmYwYWUwZGNjMTFkMmZkNzllNmZhNWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CygLe8-T5P-8gLHypKqGmv8WJJtbEy6VpMAEhU9gQ9I) why its telling me 409 error (idurar_registration_allowed) and how to tackle that

No need to really sign up.If you are using MongoDB Atlas try to directly log in with the credentials provided in the README.md file. If this too didn't work try to use your local MongoDB Compass, just replace the URL in the .env file with your localhost connection URI. Let me know your progress in both ways

**Comment by areebqazi** [comment]
Hello i ran the project through gitpod but the issue is not there . Is is fixed?

**Comment by Siddhant-Kashyap** [comment]
Yes i guess , its only to specific client , because i also has checked and its working perfectly fine

**Comment by garvitmaloo** [comment]
@Siddhant-Kashyap and @areebqazi Hii.. sorry for the late reply but yeah, there was no issue when I checked it. Everything looked fine

**Comment by inesmrad** [comment]
Maybe it's related to my browser version
